index.html

```diff
@@ -0,0 +1,137 @@
+    <title>Newsweek(Clone-2019) - News, Analysis, Politics, Business, Technology</title>
+    </head>
+    
+    <body>
+      <!-- Header Section -->
+  
+      <nav>
+
+        <section class="row bg-danger px-4 pb-2">
+          <div class="col-3 my-auto d-none d-lg-block">
+            <p class="text-white">Wed, Jan 20, 2021</p>
+          </div>
+
+          <div class="offset-lg-1 col-lg-4 col-md-3 col-sm-4 col-4">
+            <img class="img-fluid" src="./assets/newsweek-logo1.png" alt="Newsweek logo" >
+          </div>
+
+          <div class="col-lg-4 col-md-9 col-sm-8 col-8 text-right d-flex align-items-end justify-content-end my-auto">
+            <i class="d-md-none fas fa-user mr-3" style="font-size: 1.2em; color: lightgray;"></i>
+            <a href="#" class="d-md-inline d-sm-none d-none text-white mr-3">LOGIN</a>
+            <a class="btn btn-warning btn-sm rounded d-inline mr-3">SUBSCRIBE ></a>
+            <i class="d-lg-none d-md-inline fas fa-bars" style="font-size: 1.9em; color: lightgray;"></i>            
+          </div>
+        </section>
+
+        <section class="navbar navbar-expand-lg d-none d-lg-inline">
+          <ul class="navbar-nav">
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">U.S</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">World</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Business</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Tech & Science</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Culture</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Newsgeek</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Sports</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Health</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">The Debate</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold border-right border-secondary pr-3" href="nav-link">Experts</a></li>
+            <li class="nav-item mr-3"><a class="text-body font-weight-bold" href="nav-link">Vantage</a></li>
+            <li class="nav-item mr-3"><a class="text-black-50" href="nav-link">Search</a> <i class="fas fa-search"></i></li>
```

Comment:
- [ ] `<a>` anchor tag `href` attribute is used to hold ad URL or point to an `id`, it should not contain a Bootstrap class. When there is no URL or id to point at is best to use it like this within the anchor tag `href="#"`. Review all your links and make the corresponding changes.